code/__HELPERS/game.dm

```diff
@@ -556,6 +556,15 @@ block( \
 			to_chat(character, "<h2>Tag your ship with KOS to fight NT-C, SYN-C, and KOS ships</h2>")
 		else
 			return
+/**
+  * Adds the correct faction HUD to a newplayer when they connect to a shuttle
+  * mob/living/carbon/human/character -> the newplayer in question thats getting a hud
+  * obj/structure/overmap/ship/simulated/ship -> the ship the newplayer is spawning in, needed for its prefix
+  */
+/proc/AddHud(mob/living/carbon/human/character, obj/structure/overmap/ship/simulated/ship)
```

Comment:
this should be modularized, it'd be really easy just copy and paste this whole thing to `voidcrew/code/__HELPERS/game.dm` and add it to shiptest.dme if its not already tehre